pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -308,7 +308,6 @@ func (rq *ResourceQuotaController) syncResourceQuotaFromKey(key string) (err err
 	}
 	if err != nil {
 		glog.Infof("Unable to retrieve resource quota %v from store: %v", key, err)
-		rq.queue.Add(key)
```

Comment:
This is a dup of:
https://github.com/kubernetes/kubernetes/pull/57595/files#diff-ea58bce3e7609824aaacbc6149b3ecd2L255